Integration of the clearing-house

boxer-code opened the pull request:
The clearing-house folder has all necessary components to host a clearing-house additionally to the IDS-Testbed. I changed the clearing-house address of the connectors and it's tested automatically at the postman script. 
It's using the IDS-Testbed certificates and it's integrated in the compose file.
Also I added a _.gitattributes_ file to ensure that the docker files have the lf line ending.

jfernandezsqs commented:
I have reviewed this PR.
I have needed to change the following lines in the  `docker-compose.yml` file in order to work properly (Lines 121, 172 and 188):
```
context: ./Clearing-house
```
I have reviewed that the Clearing House is deployed together with the IDS-testbed but it is not using certificates provided or created by the IDS-testbed Certificate Authority. 
In detail, the certificates located at the folder `Clearing-house/docker`  (`daps_cachain.crt`) , `Clearing-house/clearing-house-app/certs` (`daps.der`) and the certificate located at the folder `Clearing-house/trusted-connector` (`consumer-keystore.p12`).

boxer-code commented:
I've updated to the latest testbed version and changed the docker-compose.yml file. To use the certificates provided by the IDS-testbed CA #97 needs to be done. The trusted connector needs certificates with appropriate subject alterative name.

jfernandezsqs commented:
I close this pull request because the implementation of FhG's Clearing House in the IDS-testbed version 1.2.0 is causing problems of interoperability and deployment of the Clearing House component.

A possible workaround will be investigated internally to be able to implement a version of FhG Clearing House at the IDS-testbed. In any case, it should be taken into account that the maintenance of this component (FhGs Clearing House) depends on its developers.
